app/views/layouts/application.html.haml

```diff
@@ -15,7 +15,7 @@
       = link_to "Social Wiki", "/", {id: 'logo'}
       %h1.flex= @title
       - if current_user
-        = link_to current_user.username, profile_path
+        = link_to current_user.username,{:controller => "sessions", :action =>"profile", :puser => current_user}
```

Comment:
:hankey: But it was working before? You probably changed the routes in a way that messed it up.